benchmark/benchmain/main.go

```diff
@@ -73,6 +75,11 @@ const (
 
 var allWorkloads = []string{workloadsUnary, workloadsStreaming, workloadsAll}
 
+type network struct {
```

Comment:
Also, how about defining the common network types in the latency package, too?

```go
var LAN = Network{Latency: ..., Kbps: ..., MTU: ...}
// etc
```

Reply:
Done.